tests/test_customer.py

```diff
@@ -16,7 +16,7 @@ def test_add_customer(client, auth, app):
         "city": "Lancaster",
         "state": "CA",
         "zip": "93536",
-        "phone_number": "6618108496",
+        "phone_number": "(661)810-8496",
```

Comment:
Shouldn't this be in the format of ###-###-####?

Reply:
All the area codes get formatted with parenthesis.

Reply:
This all looks good then.